docs/self-hosting.md

```diff
@@ -13,10 +13,66 @@ The fastest way to get your own version of Maybe running is a "one-click deploy"
 
 ## Docker
 
+0. Install docker (cf. the [docker guide](self-hosting/docker.md) for more
+   info)
+
+### Running the docker compose stack
+
+1. Create a new folder inside your working directory (eg. your home directory
+   or `/srv/docker`) named `maybe`
+
+2. Move into the `maybe` folder and copy the `docker-compose.yml` file from the
+   root of the repository
+
+3. Copy the `.env.example` file from the root of the repository and rename it
+   to `.env`
+
+4. Edit the `.env` file and adjust the `SELF_HOSTING_ENABLED` and
+   `SECRET_KEY_BASE` variables
+
+4. Run `docker-compose up -d` to bring up the stack
+
+5. Navigate to http://localhost:3000 in your browser
+
+### Running the standalone container
+
+This method might be suitable if you already have a redis and postgres service
+running and do not wish to run containers for them too.
+
+2. Copy the `.env.example` file from the root of the repository and rename it
+   to `.env`
+
+3. Edit the `.env` file and adjust the `SELF_HOSTING_ENABLED` and
+   `SECRET_KEY_BASE` variables. You might want to edit the `DB_HOST`,
+`DB_PORT`, `POSTGRES_PASSWORD`, `POSTGRES_USER` variables as well
+
+4. Run the `maybe` docker container
+```bash
+docker run -d \
+    --name app \
+    -p 3000:3000 \
+    --restart unless-stopped \
+    --env-file .env \
+    -e RUN_DB_MIGRATIONS_IN_BUILD_STEP=false \
```

Comment:
The `RUN_DB_MIGRATIONS_IN_BUILD_STEP` is specific to the Render deployments so I believe it can safely be removed here.